### DISCUSSION TITLE
With Tensorflow dropping support for Windows GPU builds, does this mean that JAX won't be able to be built on Windows with GPU support?

### DISCUSSION THREAD

**Discussion by adam-hartshorne** [jax-ml/jax#13866] [General]
As the title suggests, with the latest release of Tensorflow, support for GPU builds on Windows platforms has been removed. As JAXs currently builds using code from Tensorflow e.g. the XLA compiler code, does this mean in the future it will not be possible to build JAX for Windows platforms?

**Comment by FurkanGozukara** [comment]
ye i am not also able to use :/

**Comment by hawkinsp** [comment]
JAX depends on XLA, which may or may not have the same build policy as TensorFlow. I think that's to be determined.

It does not rule it out: I think they would accept PRs to fix problems on Windows GPU, it's just that keeping it working will be community-supported.
